builtin_interfaces/Time stamp

float32 bearing_to_wp_deg
float32 distance_to_wp_m

float32 desired_heading_deg
float32 awa_deg

bool    tacking
bool    waypoint_reached
